docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/install-and-configure-stripe-prerequisites.md

```diff
@@ -217,6 +217,100 @@ use Spryker\Zed\OauthClient\Communication\Plugin\Payment\AccessTokenPaymentAutho
 
 ```
 
+## Using Stripe in Marketplace context
+
+Install the required modules:
+
+```bash
+composer require spryker/merchant-app spryker/merchant-app-merchant-portal-gui
+```
+
+
+### config_default.php
+
+Update your `config_default.php` file with the following changes:
+
+```php
+$config[OauthClientConstants::OAUTH_PROVIDER_NAME_FOR_ACP]
+$config[OauthClientConstants::OAUTH_GRANT_TYPE_FOR_ACP]
+$config[OauthClientConstants::OAUTH_OPTION_AUDIENCE_FOR_ACP]
+$config[KernelAppConstants::TENANT_IDENTIFIER]
+```
+
+`OauthClientConstants` are replacing the deprecated `OauthClientConstants::OAUTH_PROVIDER_NAME_FOR_PAYMENT_*` constants.
+
+#### Navigation
+
+Update `config/Zed/navigation.xml` as follows:
+
+```xml
+<?xml version="1.0"?>
+<config>
+    ...
+
+    <merchant-portal-payment-settings>
+        <label>Payment Settings</label>
+        <title>Payment Settings</title>
+        <icon>payment</icon>
+        <bundle>merchant-app-merchant-portal-gui</bundle>
+        <controller>payment-settings</controller>
+        <action>index</action>
+        <pages>
+            <onboarding>
+                <label>Onboarding</label>
+                <title>Onboarding</title>
+                <icon>payment</icon>
+                <bundle>merchant-app-merchant-portal-gui</bundle>
+                <controller>payment-settings</controller>
+                <action>onboarding</action>
+                <visible>0</visible>
+            </onboarding>
+        </pages>
+    </merchant-portal-payment-settings>
+</config>
+```
+
+#### Plugins
+
+##### KernelApp
+- `\Spryker\Zed\MerchantApp\Communication\Plugin\KernelApp\MerchantAppRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
+- `\Spryker\Zed\OauthClient\Communication\Plugin\KernelApp\OAuthRequestExpanderPlugin` - Add this one to `\Pyz\Zed\KernelApp\KernelAppDependencyProvider::getRequestExpanderPlugins()`
+
+##### MessageBroker
+- `\Spryker\Zed\MerchantApp\Communication\Plugin\MessageBroker\MerchantAppOnboardingMessageHandlerPlugin` - Add this one to `\Pyz\Zed\MessageBroker\MessageBrokerDependencyProvider::getMessageHandlerPlugins()`
+- `\Spryker\Zed\KernelApp\Communication\Plugin\MessageBroker\AppConfigMessageHandlerPlugin` - Add this one to `\Pyz\Zed\MessageBroker\MessageBrokerDependencyProvider::getMessageHandlerPlugins()` (if not already present)
+
+##### AclMerchantPortal
+- `\Spryker\Zed\MerchantAppMerchantPortalGui\Communication\Plugin\AclMerchantPortal\MerchantAppMerchantPortalGuiMerchantAclRuleExpanderPlugin` - Add this one to `\Pyz\Zed\AclMerchantPortal\AclMerchantPortalDependencyProvider::getMerchantAclRuleExpanderPlugins()`
+- `\Spryker\Zed\MerchantAppMerchantPortalGui\Communication\Plugin\AclMerchantPortal\MerchantAppAclEntityConfigurationExpanderPlugin`- Add this one to `\Pyz\Zed\AclMerchantPortal\AclMerchantPortalDependencyProvider::getAclEntityConfigurationExpanderPlugins()`
```

Comment:
Can you add descriptions for each plugin?

Reply:
They are fully described in the README.md of the package where they are implemented. The PR's for them are not reviewed and not merged into main.